cmd/thriftbreak/main.go

```diff
@@ -57,8 +59,18 @@ func run(args []string) error {
 	}
 
 	lints := pass.Lints()
+	var out string
```

Comment:
This variable declared outside the loop seems unnecessary.
It's only needed inside the loop.